lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -36,22 +38,17 @@ export async function checkOnboardingBranch(
   setGitAuthor(config.gitAuthor);
   const onboardingPr = await getOnboardingPr(config);
   if (onboardingPr) {
+    logger.debug('Onboarding PR already exists');
     if (config.onboardingRebaseCheckbox) {
       handleOnboardingManualRebase(onboardingPr);
     }
-    logger.debug('Onboarding PR already exists');
-    const { rawConfigHash } = onboardingPr.bodyStruct ?? {};
-    const commit = await rebaseOnboardingBranch(config, rawConfigHash);
-    if (commit) {
-      logger.info(
-        { branch: config.onboardingBranch, commit, onboarding: true },
-        'Branch updated'
+    if (await scm.isBranchModified(config.onboardingBranch!)) {
+      invalidateExtractCache(config.baseBranch!);
```

Comment:
Do we even have an extract cache today?

Reply:
This is the `cache.scan[baseBranch]`, we always have this.
https://github.com/renovatebot/renovate/blob/03618ee581972bba8c483b0c27f92a473c45fbc6/lib/workers/repository/process/extract-update.ts#L122

Reply:
Is that done even today? Do we use it or ignore it?

Reply:
We use it to prevent extraction when possible.
https://github.com/renovatebot/renovate/blob/03618ee581972bba8c483b0c27f92a473c45fbc6/lib/workers/repository/process/extract-update.ts#L125-L136

Reply:
You mentioned it in this [comment](https://github.com/renovatebot/renovate/pull/20893#pullrequestreview-1358346515) as well.

Reply:
I thought you introduced this cache and problem in this PR

Reply:
Do we have onboarding caching set or get prior to this PR?

Reply:
Will add onboardingCache after this PT